packaging/ubuntu-trusty/debian/changelog

```diff
@@ -1,3 +1,9 @@
+amazon-ecs-init (1.20.2-1) trusty; urgency=medium
+
+  * Cache Agent version 1.20.2
+
+  -- Feng Xiong<fenxiong@amazon.com>  Thu, 30 Aug 2018 16:43:01 -0800
```

Comment:
Can you update this to include a space between your name and email? Same for the 1.20.1-1 entry as well?